### ISSUE TITLE
v10 - Accessibility issues with navigation

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#733]
## Environment
Usability testing with JAWS on Windows 10

## Problem
Pages contain multiple navigation regions all announced as "navigation". When several regions exist, they should be labelled so that they are not identical.

[Reference](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html)

**Comment by dakahn** [comment]
Tested with February DAP rulset and looks 👍🏽
